examples/react-alt/js/app.jsx

```diff
@@ -0,0 +1,176 @@
+/*jshint quotmark:false */
+/*jshint white:false */
+/*jshint trailing:false */
+/*jshint newcap:false */
+/*global React, Router*/
+var app = app || {};
+
+(function () {
+	'use strict';
+
+	app.ALL_TODOS = 'all';
+	app.ACTIVE_TODOS = 'active';
+	app.COMPLETED_TODOS = 'completed';
+	var TodoFooter = app.TodoFooter;
+	var TodoItem = app.TodoItem;
+	var TodoActions = app.todoActions;
+	var TodoStore = app.todoStore;
+
+	var ENTER_KEY = 13;
+
+	var TodoApp = React.createClass({
+		getInitialState: function () {
+			return TodoStore.getState();
```

Comment:
you could also use `[connectToStores]`(https://github.com/altjs/connect-to-stores) to wire up store data using a HOC to pass props down to the `App` component rather than directly accessing the global `TodoStore`.

This will also remove the need to listen and unlisten in `componentDidMount` and `componentDidUnmount` and is the recommended way to connect stores to your components.